dotnet/src/webdriver/WebDriver.cs

```diff
@@ -30,7 +30,7 @@ namespace OpenQA.Selenium
     /// <summary>
     /// A base class representing a driver for a web browser.
     /// </summary>
-    public class WebDriver : IWebDriver, ISearchContext, IJavaScriptExecutor, IFindsElement, ITakesScreenshot, ISupportsPrint, IActionExecutor, IAllowsFileDetection, IHasCapabilities, IHasCommandExecutor, IHasSessionId, ICustomDriverCommandExecutor, IHasVirtualAuthenticator
+    public class WebDriver : IWebDriver, ISearchContext, IJavaScriptExecutor, IFindsElement, ITakesScreenshot, ISupportsPrint, IActionExecutor, IAllowsFileDetection, IHasCapabilities, IHasCommandExecutor, IHasSessionId, ICustomDriverCommandExecutor, IHasVirtualAuthenticator, IHasDownloads
```

Comment:
Is it correct to add the downloads functionality to `WebDriver` class rather than `RemoteWebDriver`? Looks like the functionality should work with Selenium Grid, not local drivers, so `RemoteWebDriver` might be a better place for it. Or I'm missing something?

Reply:
Maybe, yes, this isn't implemented correctly anyway, so I'm overhauling. .NET interface implementation is a little different than Java's. But RemoteWebDriver probably makes more sense than the superclass.

Reply:
Yes, you were right on this, I fixed it. Thanks.